httpcore5/src/main/java/org/apache/hc/core5/http/impl/bootstrap/ServerBootstrap.java

```diff
@@ -346,6 +347,16 @@ public final ServerBootstrap addFilterLast(final String name, final HttpFilterHa
         return this;
     }
 
+    /**
+     * Create {@link RequestRouter} with LOCAL_AUTHORITY_RESOLVER (default: IGNORE_PORT_AUTHORITY_RESOLVER).
+     *
+     * @since 5.4
+     */
+    public final ServerBootstrap useLocalAuthorityResolver(final boolean localAuthorityResolver) {
```

Comment:
@cdw8j Please use a better name for the method. `#routeToLocalAuthority` for example.